src/utils/utils.js

```diff
@@ -26,4 +26,23 @@ export function sortArray(sortType, arrayToSort) {
             break;
     };
     return sortedArray;
-};
\ No newline at end of file
+};
+
+export function getRndInteger(min, max) {
```

Comment:
Lets try to use faker here as well intead of implementing this

Reply:
Updated